### PR TITLE
Added sentinel functionality to Proxy

### LINKED ISSUE #102: Redis-proxy does not have Redis sentinel support
Just like the Worker, the Proxy should also establish a sentinel connection to Redis.

### LINKED ISSUE #103: Redis sentinel group name should be an argument/environment variable

### PR DESCRIPTION
**Pull request by NicEastvillage**
Resolves #102 and resolves #103

### CODE REVIEW

File: proxy/src/main.rs [hunk 1]
```diff
@@ -75,6 +75,13 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .default_value("6379")
                 .value_name("PORT")
                 .help("Specify the redis-port to connect to"),
+        ).arg(
+            Arg::with_name("sentinel")
+                .long("sentinel")
```
Comment (cogitantium) [inline proxy/src/main.rs]:
```suggestion
                .short("m")
                .long("sentinel")
```

Add short to match worker.

Comment (NicEastvillage) [reply proxy/src/main.rs]: Solved in 5ef873ea7

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ DatScraper Proxy 0.1.0
 d502e19@aau
 
 USAGE:
-    redis-proxy [OPTIONS]
+    redis-proxy.exe [OPTIONS]
```
Comment (cogitantium) [inline README.md]:
```suggestion
    redis-proxy [OPTIONS]
```

Comment (cogitantium) [reply README.md]: Not applicable for all systems. Please remove

Comment (NicEastvillage) [reply README.md]: "redis-proxy" is not application for all systems either. Resolved in 6b7fc6f

**Pull request merged**